account/src/types.rs

```diff
@@ -1,10 +1,9 @@
-use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use namada_core::borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
```

Comment:
I find the the name `types.rs` confusing here. This seems to contain tx data.

Reply:
Yeah, this was previously in `core/src/types/transaction/account` (and there are still some data types left over in `core/src/types/transaction`), but I went with a more generic name because the crate itself doesn't have a concept of tx - it just provides some storage operations and tx integration happens in `sdk/src/tx.rs` (construction) and `tx_init_account.rs` (consumption). It would be nice if we could provide all of these from one crate, but I think we'll need more base abstractions for those